tools/scripts/src_model.mk

```diff
@@ -13,4 +13,11 @@
 SRCS += $(PROJECT)/src/main.c
 
 #Add to INCS inlcude files to be build in the porject
-INCS += $(PROJECT)/src/main.h
\ No newline at end of file
+INCS += $(PROJECT)/src/main.h
+
+#Add to SRC_DIRS directories to be used in the build. All .c and .h files from
```

Comment:
cosmetics: make sure to add a whitespace after # across all files. You currently mixed styles and sometimes there is a space, sometimes not. It just looks so much better if it's consistent...

Reply:
done